src/error/mod.rs

```diff
@@ -192,7 +179,7 @@ impl From<nom::Err<Error>> for Error {
 impl<'i> nom::error::ParseError<LocatedSpan<&'i str>> for Error {
     fn from_error_kind(span: LocatedSpan<&'i str>, _: nom::error::ErrorKind) -> Error {
         // FIXME: Add location here
```

Comment:
Is this still relevant?

Reply:
The locations for parsing errors aren't satisfactory right now, so I think we should keep this still

Reply:
I tweaked the error message so that it reflects the idea better